### PR TITLE
Add pre-commit

### PR DESCRIPTION
**Pull request by moeyensj**
... and run it on all files.

### CODE REVIEW

File: adam_core/orbits/orbits.py [hunk 1]
```diff
@@ -111,18 +111,16 @@ def to_df(
...
-            determine if any uncertainties are present and add them if so.
+            Include lower triangular covariance matrix columns.
 
         Returns
         -------
         df : `~pandas.DataFrame`
             Pandas DataFrame containing orbits.
         """
-        df = CoordinateMembers.to_df(
+        df = CoordinateMembers._to_df(
```
Comment (akoumjian) [inline adam_core/orbits/orbits.py:123]: Why explicitly call the parent class here?

File: adam_core/orbits/orbits.py [hunk 2]
```diff
@@ -169,7 +167,7 @@ def from_df(
         cls : `~adam_core.orbits.Orbits`
             Orbits class.
         """
-        data = cls._dict_from_df(
+        data = CoordinateMembers._dict_from_df(
```
Comment (akoumjian) [inline adam_core/orbits/orbits.py:170]: Same here, is there something with the inheritance that I can't remember?

**Review by akoumjian — CHANGES_REQUESTED** [review]
Approved, pending license update.

File: adam_core/orbits/orbits.py (hunk 1 above)
Comment (moeyensj) [reply adam_core/orbits/orbits.py]: At some point, there was some ambiguity as two parent classes had `.to_df` implemented but that has been fixed.

File: adam_core/orbits/orbits.py (hunk 2 above)
Comment (moeyensj) [reply adam_core/orbits/orbits.py]: Fixed.

**Comment by moeyensj** [comment]
@akoumjian Re-licensed to MIT, I've kept the THOR attribution.

**Review by akoumjian — APPROVED** [review]

**Pull request merged**